src/core/lib/security/credentials/tls/grpc_tls_credentials_options.cc

```diff
@@ -92,6 +92,19 @@ int grpc_tls_credentials_options_set_cert_request_type(
   return 1;
 }
 
+int grpc_tls_credentials_options_set_server_verification_option(
```

Comment:
Could you please add a sanity check that validates if `grpc_tls_server_authorization_check_config` is specified if `grpc_ssl_server_verification_option` is not a default value?